### PR TITLE
Bug/bookmark button fix

### PR DESCRIPTION
**Pull request by jMisch44**
## Description

Amy and I (Jennifer) fixed the bookmark button on the header bar, so that clicking on the bookmark button does not cause the page to go blank and need to be refreshed.

[video](https://www.loom.com/share/17178d5cc15248309c3340b7b49874d1)

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have removed unnecessary comments/console logs from my code
- [x] My changes generate no new warnings
- [x] I have checked my code and corrected any misspellings
- [x] No duplicate code left within changed files
- [x] Size of pull request kept to a minimum
- [x] Pull request description clearly describes changes made & motivations for said changes

### CODE REVIEW

File: src/components/common/Header.js [hunk 1]
```diff
@@ -39,13 +40,17 @@ function MainHeader(props) {
...
-  const { searchDocs } = props;
+  const { getDocs, searchDocs } = props;
   const {
     authService: { logout },
     authState,
   } = useOktaAuth();
   const { pathname } = useLocation();
 
+  const bookmarksButton = () => {
+    getDocs(authState);
```
Comment (christina-yun) [inline src/components/common/Header.js:51]: Really like how you got this to work to trigger auth state to get everything updated. It's nice that you don't have to go to the page and then have to re-refresh it after arriving which is a problem I've seen with state before.

**Review by christina-yun — APPROVED** [review]
Y'all have done a great job of putting in simple fixes for these bugs and seemingly eliminating what could be some really unwanted side effects. 

Jennifer, I also want to say that your explanation and walkthrough of the video was super helpful so that I could see the relationship between the new getDocs function and the bookmarks button.

**Review by Klove-A — APPROVED** [review]
Good job, thanks for the quick clean fix without bloating code.

**Review by dominthebox — APPROVED** [review]
Nice solution to the issue. The code looks clean, I like the refactor for the export defaults too, make the code more organized. Nice work. Feels much better to have the bookmark page loading and displaying properly.

**Comment by stu-wd** [comment]
Pretty clever fix. The only thing I can think of is maybe (once we hit MVP), we can add a quick animation to the `bookmarks` button on the header. Almost like adding something to your cart. I know the bookmark outline changes and that reinforces the user that it's been added.... but can't a man dream? Overall, good work to the both of you. I know that this was giving us fits and I'm glad it's fixed lol.

**Review by Bobby-Tav — APPROVED** [review]
The code look nice and clean, That was a clever way to fix up the bookmark button to make the call again. The video was very informative.

**Pull request merged**